app/src/main/java/com/animal_villa/MainViewModel.kt

```diff
@@ -1,4 +1,9 @@
 package com.animal_villa
+import com.animal_villa.dto.Animal
+import com.animal_villa.dto.Character
+import com.animal_villa.dto.Player
 
-class MainViewModel {
+import androidx.lifecycle.ViewModel
+
+class MainViewModel: ViewModel() {
```

Comment:
Use viewModel to prepare data for UI https://developer.android.com/topic/libraries/architecture/viewmodel. They are scoped to the lifecycle